### PR TITLE
Token

### PR DESCRIPTION
**Pull request by tarasIV**

### CODE REVIEW

File: src/app/login/login/login.component.ts [hunk 1]
```diff
@@ -27,8 +27,9 @@ export class LoginComponent implements OnInit {
         this.router.navigate(['/shell/admin-panel/']);
       },
         error => {
-          console.log(error);
-          this.errorMessage = error.error.status.message;
+          if (error.error.status.message) {
+          this.errorMessage = 'Ви ввели невірні дані';
```
Comment (dmitryy90) [inline src/app/login/login/login.component.ts]: indentation

File: src/app/services/admingroups.service.ts [hunk 2]
```diff
@@ -5,20 +5,15 @@ import { Group } from '../models/group.model';
...
 import 'rxjs/add/operator/catch';
-import { forkJoin } from 'rxjs';
 import { isString, isNumber } from 'util';
 
-
-
 @Injectable()
 export class AdmingroupsService {
 
-  private ClassesUrl = 'https://fierce-shore-32592.herokuapp.com/classes';
```
Comment (dmitryy90) [inline src/app/services/admingroups.service.ts]: needs to be changed across the app

File: src/app/login/login/login.component.ts (hunk 1 above)
Comment (tarasIV) [reply src/app/login/login/login.component.ts]: Fixed

File: src/app/services/admingroups.service.ts (hunk 2 above)
Comment (tarasIV) [reply src/app/services/admingroups.service.ts]: Replaced

File: src/app/services/admin-subjects.service.ts [hunk 3]
```diff
@@ -12,16 +12,16 @@ export class AdminSubjectsService {
...
   public postSubject(subject: Subject): Observable<Subject> {
     return this._http
-      .post<any>(this.url, subject)
+      .post<any>('subjects', subject)
       .map(response => response.data);
   }
 
   public putSubject(subject: Subject): Observable<Subject> {
-    return this._http.put<any>(`${this.url}/${subject.subjectId}`, subject).map(response => response.data);
+    return this._http.put<any>(`subjectss/${subject.subjectId}`, subject).map(response => response.data);
```
Comment (Rihovardini) [inline src/app/services/admin-subjects.service.ts]: subjects instead of subjectss

Comment (tarasIV) [reply src/app/services/admin-subjects.service.ts]: Missclick! Deleted

**Pull request merged**